src/python/bot/fuzzers/libFuzzer/engine.py

```diff
@@ -270,9 +271,14 @@ def reproduce(self, target_path, input_path, arguments, max_time):
       max_time: Maximum allowed time for the reproduction.
 
     Returns:
-      A Result object.
+      A tuple of (return_code, output).
     """
-    raise NotImplementedError
+    runner = libfuzzer.get_runner(target_path)
+    launcher.set_sanitizer_options(target_path)
```

Comment:
it is weird to have set_sanitizer_options in launcher ? should it be in engine_common or something.

Reply:
I guess the assumption is that different fuzzing engines may need to overwrite some of the default values? What would happen if e.g. AFL launcher wouldn't call this method? Will the default values be set up by any other code?

Reply:
I can move this later in another PR. I think it makes sense to share this for all engines, and they can overwrite things if need be.